sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/LiveVariableAnalysis/CSharp/CSharpLiveVariableAnalysis.cs

```diff
@@ -187,6 +191,28 @@ public sealed class CSharpLiveVariableAnalysis : AbstractLiveVariableAnalysis
             }
         }
 
+        private void ProcessLocalFunction(SyntaxNode instruction, HashSet<ISymbol> assignedInBlock,
```

Comment:
Although there are unfortunately no direct unit test at the moment for LVA, I believe it would be a good thing to test just these changes in a dedicated UT. You could use something similar to what is in `ExplodedGraphTest`